.github/workflows/lint_pr.yml

```
@@ -7,7 +7,11 @@ on:
       - edited
       - synchronize
 
-jobs:
+  permissions:
+  pull-requests: write
+  statuses: write
+
+  jobs:
```

should this not be on the same level as before?
```suggestion
jobs:
```

---

Thank you.
Fixed.